### DISCUSSION TITLE
PP change isn't really shown

### DISCUSSION THREAD

**Discussion by Feodor0090** [ppy/osu#21825] [Q&A] [answered]
![изображение](https://user-images.githubusercontent.com/53872073/209714372-30afbbc8-ac83-492a-a468-df88e2a3f445.png)
I had 56pp before the play. Difference isn't shown.
Logs jic: 
[network.log](https://github.com/ppy/osu/files/10310369/network.log)
[runtime.log](https://github.com/ppy/osu/files/10310370/runtime.log)

(also - is it okay that i have 58 total while top 3 plays are 58 40 37? Formula changed or something is broken?)

**Comment by peppy** [comment, marked as answer]
This should be fixed now! Please try again.
